src/circllhist.c

```diff
@@ -95,15 +95,7 @@ struct hist_flevel {
   uint8_t l2;
   uint8_t l1;
 };
-struct histogram {
```

Comment:
histogram is a private struct is isn't exposed.  We don't want to drop out pants here.. this should stay in the implementation, not in the header.